doc/sphinx/conf.py

```diff
@@ -183,18 +183,18 @@ def setup(app):
 nitpicky = True
 
 nitpick_ignore = [ ('token', token) for token in [
-    'tactic',
-    # 142 occurrences currently sort of defined in the ltac chapter,
-    # but is it the right place?
-    'module',
-    'redexpr',
-    'modpath',
-    'dirpath',
     'collection',
+    'command',
+    'dirpath',
+    'modpath',
+    'module',
+    'red_expr',
+    'symbol',
+    'tactic',
+    'term_evar',
```

Comment:
`term_evar` should not appear here anymore if it is defined in the Gallina extensions chapter.